native/android/app/build.gradle

```diff
@@ -90,8 +90,8 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 23
```

Comment:
The downgrade is on purpose, it has been done as part of the RN upgrade here https://github.com/Skyscanner/backpack/commit/da1e519011ccedbcb7e8754d07f535d398532d61 but it was a "nice to do" not a requirement for the RN upgrade